theoplayer/how-to-guides/ios/millicast/getting-started.mdx

```diff
@@ -0,0 +1,72 @@
+# Getting started with Millicast for iOS
+
+import Intro from '../../shared/millicast/_getting-started-intro.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Intro />
+
+## Usage
+
+1. Follow [our Getting Started guide](../../../getting-started/01-sdks/03-ios/00-getting-started.md)
+   to set up THEOplayer in your iOS app.
+2. Add the `THEOplayer-Integration-Millicast` dependency to your project.
+3. Add the `MillicastIntegration` to the player.
```

Comment:
Rename `MillicastIntegration` to `THEOplayerMillicastIntegration` as that's the product name.